### PR TITLE
Chat api 2.11

### PR DESCRIPTION
**Pull request by coders-kitchen**
Completes the release of version 2.11

### CODE REVIEW

File: chat-api/v2/openapi.yaml [hunk 1]
```diff
@@ -2048,7 +2094,7 @@ components:
           enum:
             - text
             - url
-          description: What kind of payload should be sent
+          description: What kind of payload should be send
```
Comment (Lennonka) [inline chat-api/v2/openapi.yaml:2097]: This was correct before.

File: chat-api/v2/README.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+#### WhatsApp : Location with URL
+
+When a user sends a location to the business, the URL of the origin site is present as well.
+
+#### WhatsApp : WhatsApp business account id (WABA ID)
+
+The account model of WhatsApp transports the WhatsApp business account id (WABA ID) as well.
+
+This id is generated by WhatsApp, when an account is created. 
+
```
Comment (Lennonka) [inline chat-api/v2/README.md:31]: Tiny improvements suggestions:

#### Bulk messaging

Beginning with this release, the Chat API supports bulk messaging.

Bulk messaging enables a business to send multiple messages in order to a single user without the need to wait for
delivery receipts for individual messages.

Between 2 and 20 messages can be chained on all supported channels - SMS, WhatsApp, and Viber.

All content types that a specific channel supports are supported.

**Note** On WhatsApp the 24-hour support window policy is still effective when using this feature.

#### WhatsApp - Location with a URL

When a user sends a location to the business, the URL of the origin site is present as well.

#### WhatsApp - WhatsApp business account id (WABA ID)

File: chat-api/v2/openapi.yaml [hunk 3]
```diff
@@ -263,6 +260,36 @@ paths:
...
         default:
           $ref: '#/components/responses/Default'
+  /bulks:
+    post:
+      operationId: sendBulkMessage
+      summary: Send a bulk of messages
+      tags:
+        - "Send messages"
+      description: |
+        Send a bulk of messages via one channel to one receiver.
```
Comment (Lennonka) [inline chat-api/v2/openapi.yaml:270]: receiver -> recipient

File: chat-api/v2/openapi.yaml [hunk 4]
```diff
@@ -263,6 +260,36 @@ paths:
...
+
+        The system guarantees that the messages are send in order to the receiver
```
Comment (Lennonka) [inline chat-api/v2/openapi.yaml:272]: sent :)

File: chat-api/v2/openapi.yaml [hunk 5]
```diff
@@ -2075,6 +2121,14 @@ components:
       type: object
       description: |
         Viber message to be sent
+
+        The following content / service type combination sare valid
```
Comment (Lennonka) [inline chat-api/v2/openapi.yaml:2125]: A space in a bad place

File: chat-api/v2/samples/sending_bulk_messages.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+# Sending bulk messages
+
+This is an example request for sending bulk messages.
+
+Bulk messages are multiple messages sent, in order, to one user from one sender.
+
+Every channel supported by Chat API is also supported on the bulk messaging
```
Comment (Lennonka) [inline chat-api/v2/samples/sending_bulk_messages.md:7]: supported by the bulk messaging

File: chat-api/v2/samples/rich_media_template_with_text.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
+# Rich Media Template with text
+
+This is an example request for sending a text together with a rich media template
```
Comment (Lennonka) [inline chat-api/v2/samples/rich_media_template_with_text.md:3]: Add a trailing period after the sentence (watch it in the following places as well)

File: chat-api/v2/openapi.yaml (hunk 4 above)
Comment (coders-kitchen) [reply chat-api/v2/openapi.yaml:272]: thank you :)

**Pull request closed without merging**